### ISSUE TITLE
two encoders at once?

### ISSUE THREAD

**Issue by troylball** [adafruit/seesaw#54]

**Comment by troylball** [comment]
Hi I am looking for some sample python code that will allow two encoders at the same time.  I have already solders the back to change one to a different i2c #,  any help would be appreciated

**Comment by troylball** [comment]
The rotary encoder qt board 4991.  I have it running with one and I soldered the back to get a second i2c # , but my limited tweeks did not work ☹

https://learn.adafruit.com/adafruit-i2c-qt-rotary-encoder/overview
Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by ladyada** [comment]
do you mean circuitpython

**Comment by troylball** [comment]
Yes I guess I did,    I am using the code from  https://learn.adafruit.com/adafruit-i2c-qt-rotary-encoder/overview wit a pi Zero 2 w and maybe a pi 4.


But that gest me wondering is there just regular python code as  well?   If so I would like to see that as well.

Cheers,
Troy




Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
